dojo/unittests/tools/test_pmd_parser.py

```diff
@@ -0,0 +1,24 @@
+from django.test import TestCase
+from dojo.tools.pmd.parser import PmdParser
+from dojo.models import Test
+
+
+class TestPMDParser(TestCase):
+
+    def test_parse_file_with_no_vuln_has_no_findings(self):
+        testfile = open("dojo/unittests/scans/pmd/pmd_no_vuln.csv")
+        parser = PmdParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(0, len(findings))
+
+    def test_parse_file_with_one_vuln_has_one_findings(self):
+        testfile = open("dojo/unittests/scans/pmd/pmd_one_vuln.csv")
+        parser = PmdParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(1, len(findings))
+
+    def test_parse_file_with_multiple_vuln_has_multiple_finding(self):
+        testfile = open("dojo/unittests/scans/pmd/pmd_many_vulns.csv")
+        parser = PmdParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(15, len(findings))
```

Comment:
Could you add more check on attributes of the result?
Like:
```
self.assertEqual("<foo>", findings[0].severity)
```

Reply:
I think i am not familiar with that, so I feel i am not confident.

Reply:
it's ok. no pb. will try to add it today.

Reply:
Great. Thanks.
To pass a PR, it is too difficult. :(

Reply:
no, you did a great job !

Reply:
added some checks

Reply:
Yeah, I checked.
Great.
Thanks for your help. 👍